project/Component/nef/Instances/MacPlaygroundSystem.swift

```diff
@@ -0,0 +1,112 @@
+//  Copyright © 2019 The nef Authors.
+
+import Foundation
+import NefCommon
+import Bow
+import BowEffects
+
+class MacPlaygroundSystem: PlaygroundSystem {
+    private let fileManager = FileManager.default
```

Comment:
Functions in `PlaygroundSystem` should return `EnvIO<FileSystem, E, A>` and pass the file system as a parameter, so that we can easily change it in testing. This dependency is hardcoded  here and therefore makes testing more difficult.

Reply:
Also, if not all functions require a `FileSystem`, consider segregating the functionalities into several protocols, as they probably do not belong together.

Reply:
we will work on in a future PR